DESIGN_SYSTEM.md

```
@@ -0,0 +1,140 @@
+# CS Dashboard – Design System
+
+> M1 – FE Design Prototype. Preview tokens at `/design-tokens`.
+
+## Color Palette
+
+### Primary – Indigo
+| Token | Value | Usage |
```

markdownlint-cli2 (0.22.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)